unixfs/src/pb/unixfs.rs

```diff
@@ -0,0 +1,185 @@
+// Modified automatically generated rust module for 'unixfs.proto' file
+#![allow(rust_2018_idioms)]
+#![allow(non_snake_case)]
+#![allow(non_upper_case_globals)]
+#![allow(non_camel_case_types)]
+#![allow(unused_imports)]
+#![allow(unknown_lints)]
+#![allow(clippy::all)]
+#![cfg_attr(rustfmt, rustfmt_skip)]
+use std::io::Write;
+use std::borrow::Cow;
+use quick_protobuf::{MessageRead, MessageWrite, BytesReader, Writer, WriterBackend, Result};
+use std::convert::TryFrom;
+use std::ops::Deref;
+use std::ops::DerefMut;
+use quick_protobuf::sizeofs::*;
+use super::*;
+#[derive(Debug, Default, PartialEq, Clone)]
+pub struct Data<'a> {
+    pub Type: mod_Data::DataType,
+    pub Data: Option<Cow<'a, [u8]>>,
```

Comment:
did `clippy`/`fmt` not complain about `Type` and `Data` starting with an uppercase char?

Reply:
This is generated by pb-rs ... It doesn't really attempt to get to nice rust field names, at least the version I used here didn't.